src/main/java/org/jenkinsci/plugins/codesonar/CodeSonarPublisher.java

```diff
@@ -455,28 +518,11 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
                 }
             }
         }
-
+   
```

Comment:
is this extra whitespace or something?

Reply:
no, but I removed it to avoid extra stuff in diff.

Reply:
Err, well, maybe I removed the wrong thing and my local diff program doesn't fuss over whitespace so I didn't notice it until now.  Either way, it doesn't matter.